.github/workflows/conda-forge-install-and-test.yaml

```diff
@@ -0,0 +1,48 @@
+name: Install all deps from conda-forge and run pytest
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'release/*'
+    tags:
+      - 'v*'
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    env:
+      DISPLAY: ':99.0'
+      OS: ${{ matrix.os }}
+
+    steps:
+    - name: checkout source code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: '0'
+    - name: setup ubuntu-latest xvfb
+      uses: ./.github/actions/setup-ubuntu-latest-xvfb
+      if: runner.os == 'Linux'
+    - name: install dependencies
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        activate-environment: qcodesforge
+        environment-file: .github/static/environment_forge_full.yml
+        auto-activate-base: false
+    - name: generate test dbs
+      uses: ./.github/actions/generate-test-dbs
+      # todo is it an issue that this pip installs qcodes with deps?
```

Comment:
I think we should pull the pip install logic out of that script and simply require that the user has installed qcodes editable before running this.

Reply:
or we create a job that creates the DBs, puts then as some artifact and passes that artifact to this job :)

Reply:
@astafan8 Indeed that would be better but that is out of scope for this pr

Reply:
Furthermore it is not obvious that you can share data between jobs that are defined in different workflow files https://docs.github.com/en/actions/guides/storing-workflow-data-as-artifacts#passing-data-between-jobs-in-a-workflow